integration/kubernetes/init.sh

```diff
@@ -40,6 +40,31 @@ wait_pods_ready()
 	done
 }
 
+build_custom_stress_image()
+{
+	echo "Build custom stress image"
+	registry_image="docker.io/library/registry:2"
+	arch=$("${SCRIPT_PATH}/../../.ci/kata-arch.sh")
+	if [[ "${arch}" == "ppc64le" || "${arch}" == "s390x" ]]; then
+		# that image is not built for these architectures
+		registry_image="docker.io/ibmcom/registry:2.6.2.5"
+	fi
+
+	runtimeclass_files_path="${SCRIPT_PATH}/runtimeclass_workloads"
+	pushd "${runtimeclass_files_path}/stress"
+	[ "${container_engine}" == "docker" ] && sudo -E systemctl restart docker
+	sudo -E "${container_engine}" build . -t "${stress_image}"
+	popd
+
+	if [ "${stress_image_pull_policy}" == "Always" ]; then
+		echo "Store custom stress image in registry"
+		sudo -E "${container_engine}" run -d -p ${registry_port}:5000 --name "${registry_name}" "${registry_image}"
```

Comment:
What happens if this fails? Do we continue to run the CI?

Reply:
When this fails, the CI run exits with an error.

Reply:
great, thanks